src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -84,52 +86,46 @@ public ProcessStats GetStats()
 
         private void OnErrorDataReceived(object sender, DataReceivedEventArgs e)
         {
-            // TODO: per language stdout/err parser?
             if (e.Data != null)
             {
-                string msg = e.Data;
-                if (msg.IndexOf("warn", StringComparison.OrdinalIgnoreCase) > -1)
+                ParseConsoleLog(e.Data, true);
+            }
+        }
+
+        internal void ParseConsoleLog(string msg, bool errorData = false)
+        {
+            if (msg.IndexOf("warn", StringComparison.OrdinalIgnoreCase) > -1)
+            {
+                BuildAndLogConsoleLog(msg, LogLevel.Warning);
```

Comment:
I think there's going to be a change in behavior where a user's logs containing the word "warn" will now be a warning instead of an "info" log, but I think that's good :)

Reply:
Ahh wait sorry - should this only be the case if only if `errorData == true`? Otherwise, any user log that contains the message "warn" would be marked with warning, which I don't think is necessarily a valid assumption (unless java)?

Reply:
you are right. Updated and also added test.